examples/nullable.aps

```diff
@@ -13,7 +13,7 @@ module NULLABLE[T :: var GRAMMAR[]] extends T begin
   circular attribute Items.items_nullable : OrLattice;
   pragma synthesized(items_nullable);
 
-  circular attribute Grammar.grammar_nullable : NullableTable;
```

Comment:
Here also.  I expect we need this to be circular.  Maybe it's being lost because of fibering?